lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/jdo/EntityNameValidation.java

```diff
@@ -17,7 +17,7 @@
 
 	// match one or more whitespace characters
 	private static final Pattern ALLOWABLE_CHARS = Pattern
-			.compile(ModelConstants.VALID_ENTITY_NAME_REGEX);

```

Comment:
put back in ModelConstants, otherwise the change will break the GWT portal

Reply:
Comment the ModelConstants class, explaining that the purpose is to define constants used both in the back end and in the GWT portal.